challenges/20200602/billy/script.js

```diff
@@ -0,0 +1,20 @@
+let app = new Vue({
+    el: "#vueplayground",
+    data: {
+        seen: false,
+        list: [
+            { text: 'First Message Ever' },
+        ],
+
+        message: ""
+    },
+    methods: {
+        add: function () {
+            if (!(this.message == "" || this.message.trim() == "")) {
```

Comment:
1. Checking if two variables are equal in the language of Javascript uses ```===``` as a more strict operator instead of ```==```. Please change the operator in the ```if``` statement.
2. ```this.message.trim() === ""``` already includes the case of ```this.message === ""```